app/src/main/java/com/brandio/androidsample/InterstitialActivity.java

```diff
@@ -57,6 +58,7 @@ private void loadAd() {
         Placement placement;
         try {
             placement = Controller.getInstance().getPlacement(placementId);
+            placement.getData();
```

Comment:
What is placement.getData()?

Reply:
It seems like redundant line) Will clear it.